src/content/docs/experiment/link-element.mdx

```diff
@@ -0,0 +1,141 @@
+
+Second, your site visitor’s [Web Monetization agent](#web-monetization-agent) could be coded to handle multiple links in a particular way. For example, one visitor’s Web Monetization agent may split payments evenly between all links while another Web Monetization agent only sends payments to the first monetization link it finds.
+
+:::tip[Recommendation]
+The Web Monetization agent built into the <LinkOut href="https://github.com/interledger/web-monetization-extension">Interledger Foundation's extension</LinkOut> splits payments evenly between all links. We recommend other extension developers follow this approach.
+:::
+
+### Iframes
+
+Nested browsing contexts (iframes) can contain monetization `<link>` elements; however, your site visitor's [Web Monetization agent](#web-monetization-agent) determines how iframes are monetized. A few examples of how a Web Monetization agent can monetize iframes include:
+
+* Splitting payments evenly between all monetization links within both the parent and the iframe
+* Splitting payments evenly between all monetization links in the parent and the first monetization link it finds in the iframe's `<head>`, while ignoring any other monetization links in the iframe
+    :::note
+    This is the approach used in the <LinkOut href="https://github.com/interledger/web-monetization-extension">Interledger Foundation's extension</LinkOut>.
+    :::
+* Sending payments to the first monetization link it finds in the parent browsing context and ignoring all other monetization links in the parent and the iframe
+
+### Audio, video, and picture elements
+
+The following HTML elements can be web monetized by adding the `<link>` element between the open and close tags.
+
+<Tabs>
+    <TabItem label="<audio>" icon="seti:audio">
+        ```html
+        <audio src="music.mp3">
+            <link rel="monetization" href="https://wallet.example.com/bob">
+        </audio>
+        ```
+    </TabItem>
+    <TabItem label="<video>" icon="seti:video">
+        ```html
+        <video src="myvideo">
+            <link rel="monetization" href="https://wallet.example.com/bob">
+        </video>
+        ```
+    </TabItem>
+    <TabItem label="<picture>" icon="seti:image">
+        ```html
+        <picture srcset="cat.jpeg">
+            <link rel="monetization" href="https://wallet.example.com/bob">
+        </picture>
+        ```
+    </TabItem>
+</Tabs>
+
+## How it works
+
+### HTML DOM API
+
+Due to the efforts of the <LinkOut href="https://www.w3.org/">W3C</LinkOut>, there are a number of APIs that all major browser vendors build into their browsers. One such API is the <LinkOut href="https://developer.mozilla.org/en-US/docs/Web/API/HTML_DOM_API">HTML DOM API</LinkOut>, which defines the functionality of HTML elements.
+
```

Comment:
This maybe not very relevant to this page? Maybe just link DOM API in next paragraph?